login.css

```diff
@@ -1,5 +1,6 @@
 /* Import  font from google fonts*/
-@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@300;500&display=swap');
```

Comment:
15/08/2021: I trailing some font and the font I decided to use are Roboto and source sans pro. I import font from google fonts. The font for normal text is Roboto and the fonts for heading will be source sans pro.